test/test.js

```diff
@@ -87,3 +91,54 @@ describe('Database Helpers', () => {
 
 });
 
+describe('Request', () => {
+  describe('GET /details/:propertyId', () => {
+
+    test('should return an object with the expected properties', (done) => {
+      return request(app)
+        .get('/details/1')
+        .then(response => {
+          expect(response.body).toHaveProperty('typeOfPlace');
+          expect(response.body).toHaveProperty('propertyType');
+          expect(response.body).toHaveProperty('cancellation');
+          expect(response.body).toHaveProperty('houseRules');
+          expect(response.body).toHaveProperty('about');
+          expect(response.body).toHaveProperty('sleepingArrangements');
+          expect(response.body).toHaveProperty('propertyId');
```

Comment:
maybe we could implement a loop to generate all these expect statements for future tests you make think it would be cool and less typing for you

Reply:
making a ticket for this one. good call out. i had the same thought but was feeling lazy